parsing/tools/build.gradle.kts

```diff
@@ -0,0 +1,52 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to you under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+plugins {
+    java
+    kotlin("jvm")
+}
+
+dependencies {
+    api(project(":parsing:dialects:bigquery"))
```

Comment:
Just a note, once https://github.com/googleinterns/calcite/pull/211 gets merged these dependencies will need to be changed to :parsing:intermediate:...

Reply:
Thanks for the heads up, I'll change it once 211 gets merged since that'll probably be merged first.